### PR TITLE
Menu to let the user change the servers ip

### PR DESCRIPTION
**Pull request by EnterpriseFreak**
I've added a menu to let the user enter a custom ip address so that that there is no longer a need for the hardcoded one. Also i added code to underclock the cpu down to 333 MHz so that the battery life is improved.

### CODE REVIEW

**Comment by WerWolv** [comment]
Nice! Thank you :)
For the message that get's displayed, it's not the IP the switch should connect to, it's the IP the UPD broadcast gets sent to so the PC and the switch will automatically connect. So you should make all parts of the IP configurable but have it default on 192.168.1.255

**Comment by EnterpriseFreak** [comment]
No problem!
I have changed the code so that you can edit the whole IP and set 192.168.1.255 as a default just like you recommended. I this helps you!

**Comment by EnterpriseFreak** [comment]
So i changed the text for the IP menu so that it is more helpful and also made the code a bit better to read. If everything is ok for you would you mind to merge my modifications?

File: source/main.c [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+    printf("Please set up the ip you would like to send the UDP broadcast too\n.");
+    printf("Usally this address is 192.168.X.255 where X is the 3rd block of your local IP.\n");
+    printf("If UDP broadcasting does not work for you, use the IP of your computer instead.\n");
+
+	consoleUpdate(NULL);
+
+	pcvSetClockRate(PcvModule_Cpu, CPU_CLOCK); //Underclock the CPU
+	appletSetScreenShotPermission(0); //Disable the screenshot function because it is not needed for the program
+	appletBeginBlockingHomeButton(0); //Block the Home Button to prevent acidental use. Mainly useful for the pro controller since (+) and Home are close together.
```
Comment (EnterpriseFreak) [inline source/main.c]: Have to remove that one. Will lock the console if it is ran from the hbmenu nsp.

**Comment by WerWolv** [comment]
Okay. I'll merge it once that has been resolved. Thanks a lot :)

**Comment by EnterpriseFreak** [comment]
All done!
I also added a last minute patch so that you can also push the XBOX Button without the touchscreen. Just press R+LSTICK and it sends a XBOX press. This is useful when the switch is docked.

**Comment by WerWolv** [comment]
Thanks!

**Pull request merged**